### PR TITLE
523 Email confirmation

### PR DESCRIPTION
**Pull request by aditiapratama1231**
#523 

### Breakdown:
- [x] Create confirmed field
- [x] check if user not confirmed they can't logged in
- [x] except admin

### Screenshot:
table user who have confirmed
user confirm

if logged in as user with no have confirmed
not confirm

if logged in as user with have confirmed
confirm

if logged in as admin, note : admin doesn't have confirmed, and they can logged in
admin confirm

### CODE REVIEW

File: app/controllers/user_authorization_controller.py [hunk 1]
```diff
@@ -42,9 +42,12 @@ def login(request):
                         if user.as_dict()['role_id'] != 1 and user.as_dict()['role_id'] != 8:
                             return BaseController.send_error_api({'unauthorized': True}, 'unauthorized, must be admin to access this page.')
                     if user.verify_password(password):
-                        token = userservice.save_token()
-                        user = userservice.include_role_data(user.include_photos().as_dict())
-                        return BaseController.send_response_api({'access_token': token['data'].access_token, 'refresh_token': token['data'].refresh_token}, 'User logged in successfully', user)
+                        if user.as_dict()['confirmed'] is not None or user.as_dict()['role_id'] == 1:
```
Comment (andy-shi88) [inline app/controllers/user_authorization_controller.py]: I think we can simplify this without using as_dict() here, just use user.confirmed should do.

**Review by andy-shi88 — CHANGES_REQUESTED** [review]

File: app/controllers/user_authorization_controller.py (hunk 1 above)
Comment (aditiapratama1231) [reply app/controllers/user_authorization_controller.py]: okay mr @andy-shi88  fixed

**Review by andy-shi88 — APPROVED** [review]
cool @aditiapratama1231

**Comment by andy-shi88** [comment]
I'll merge this along with the email confirmation message feature, let's put this here first @aditiapratama1231

**Pull request closed without merging**